cobrawap/pipeline/stage05_wave_characterization/configs/config_template.yaml

```diff
@@ -7,7 +7,7 @@ STAGE_NAME: 'stage05_wave_characterization'
 PROFILE: 'dataset_key'
 
 # Name of the output file
-STAGE_OUTPUT: 'wave-wise_measures.csv'
+STAGE_OUTPUT: 'wave_wise_measures.csv'
```

Comment:
I don't think this should be changed. We would need to evaluate if this is helpful or more consistent in any way.

Reply:
We previously changed the stage name, by using "channel_wave" instead of "channel-wave". So I was acting here coherently.

Reply:
see comment above